tests/test_frontier.py

```diff
@@ -215,6 +215,7 @@ def test_resume_job():
     site = list(frontier.job_sites(job.id))[0]
 
     assert job.status == 'ACTIVE'
+    assert job.stop_requested is None
```

Comment:
I don't think job.stop_requested is ever set, so this doesn't really test anything? Correct me if I'm wrong

Reply:
You are right, it was just a assert to make sure that however the code evolve in the future it will just break the test if somehow the stop_request is not None after a resume. xD

I updated the test using the same pattern that it was being used before, but now it simulates a stop request.